index.js

```diff
@@ -66,7 +66,7 @@ function handler(opts) {
 
 			if (!res.headersSent) {
 				if (opts.onTimeout) {
```

Comment:
Same here...

Reply:
`opts` is always defined - see  https://github.com/debitoor/adminux/commit/1cbb700a2840e6c62e5aca397d0ef6f526318967.

If no `timeout()` is supplied in opts, then this if will not be entered - but it will not throw any errors.

Reply:
woops @hleote , I pasted wrong link :-) Updated my comment